packages/core/src/browser/keys.ts

```diff
@@ -127,7 +127,20 @@ export namespace KeySequence {
                 if (previous) {
                     keyCodeResult += separator;
                 }
-                keyCodeResult += Key.getEasyKey(keyCode.key).easyString.toUpperCase();
+
+                const keyString = Key.getEasyKey(keyCode.key).easyString;
+
+                if (keyCode.key.keyCode >= Key.KEY_A.keyCode && keyCode.key.keyCode <= Key.KEY_Z.keyCode ||
+                    keyCode.key.keyCode >= Key.F1.keyCode && keyCode.key.keyCode <= Key.F2.keyCode) {
```

Comment:
`<= Key.F2` seems to be bogus?

Reply:
Well spotted @akosyakov, it was supposed to be `F24` so things like `f11` are shown as `F11`